File: cs/src/AutoDiff/Compiled/Reification.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace AutoDiff.Compiled
{
	class Reification : TapeElement
    {
        public double Min;
        public double Max;
		public int Condition;
		public int NegatedCondition;

        public override void Accept(ITapeVisitor visitor)
        {
            visitor.Visit(this);
        }
	}
}

File: java/src/main/java/AutoDiff/Compiled/Reification.java
//
// Translated by CS2J (http://www.cs2j.com): 12.11.2016 09:40:32
//

package AutoDiff.Compiled;

public class Reification  extends TapeElement
{
    public double Min;
    public double Max;
    public int Condition;
    public int NegatedCondition;
    public void accept(ITapeVisitor visitor) throws Exception {
        visitor.visit(this);
    }

}


